examples/package.json

```diff
@@ -15,10 +15,11 @@
     "@react-page/plugins-video": "0.0.0",
     "@react-page/renderer": "0.0.0",
     "@react-page/ui": "0.0.0",
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3",
-    "@material-ui/core": "3.6.1",
-    "@material-ui/icons": "3.0.1",
+    "react": "link:../node_modules/react",
+    "react-dom": "link:../node_modules/react-dom",
+    "@material-ui/core": "4.3.0",
+    "@material-ui/icons": "4.2.1",
+    "@material-ui/styles": "4.3.0",
```

Comment:
since we have a package-lock.json, we could relax the version here (with ^)

Reply:
I will try to solve the pipeline issue and will allow also higher versions for the material ui dependecies today.

Reply:
maybe because of a version missmatch in package-lock.json or yarn.lock?